src/util/send.ts

```diff
@@ -15,22 +14,12 @@ const messageToUserId = new LimitedSizeMap<
 
 export const DELETE_EMOJI = '🗑️';
 
-export async function getResponseChannel(message: Message) {
```

Comment:
This was used to avoid posting messages in #help-typescript, since forum threads don't have an equivalent, it doesn't serve a purpose anymore.